src/components/TodoList.jsx

```diff
@@ -1,14 +1,16 @@
 import { useSelector, useDispatch } from "react-redux";
-import { deleteTodo, completedTodo } from "../redux/action/actions";
+import { completedTodo, deleteTodo } from "../redux/todosSlice";
 
 const TodoList = () => {
-  let allTodos = useSelector((state) => state.todos.todos);
+
+  const allTodos = useSelector((state) => state.todos);
   const dispatch = useDispatch();
 
+  if (allTodos.length == 0) return "No todos on your list";
```

Comment:
If there are no todos available it would be better to return a valid React element instead of returning just a string

Reply:
yeap. it is fixed!